### PR TITLE
Add AppSet details in 2.5

### PR DESCRIPTION
**Pull request by oafischer**
https://github.com/stolostron/backlog/issues/23963

### CODE REVIEW

File: applications/app_model.adoc [hunk 1]
```diff
@@ -69,7 +69,9 @@ Placement rules (`placementrule.apps.open-cluster-management.io`) define the tar
 [#applicationset]
 == ApplicationSet
 
-`ApplicationSet` is a sub-project of Argo CD that is supported by the GitOps Operator. `ApplicationSet` adds multicluster support for Argo CD applications. You can create an `ApplicationSet` from the {product-title-short} console. 
+`ApplicationSet` is a sub-project of Argo CD that is supported by the GitOps Operator. `ApplicationSet` adds multicluster support for Argo CD applications. You can create an `ApplicationSet` from the {product-title-short} console.
```
Comment (dockerymick) [inline applications/app_model.adoc]: You can create an application set from the {product-title-short} console.

File: applications/app_model.adoc [hunk 2]
```diff
@@ -69,7 +69,9 @@ Placement rules (`placementrule.apps.open-cluster-management.io`) define the tar
...
+
+**Note:** For more details on the prerequisites for deploying `ApplicationSet`, see xref:../applications/gitops_config.adoc#register-gitops[Registering managed clusters to GitOps].
```
Comment (dockerymick) [inline applications/app_model.adoc:74]: In the other PRs I have been seeing the following:

**Note**:

**Review by dockerymick — APPROVED** [review]
Left a couple of comments, but they are small updates.

File: applications/app_model.adoc (hunk 2 above)
Comment (dockerymick) [reply applications/app_model.adoc:74]: Maybe that change was in a previous version im not sure. keep it as is. When i did a search in VSC, there were only a couple of instances where the colon was not also bolded. This is super small, but can be consistent. Again, keep the bold as is

Comment (oafischer) [reply applications/app_model.adoc:74]: Chris helped me out on that once, the colon is always supposed to be bold but there are several instances throughout the doc where it is not bold. We can keep an eye out for those as we work on stuff and maybe fix them on the side in our PRs. I also clarified this in our [internal docs](https://stolostron.github.io/rhacm-docs/wr_special_tag_guidance) for future reference.

**Pull request merged**